doc/getting_started/toolchain_other_x_compilers.rst

```diff
@@ -29,6 +29,16 @@ Follow these steps to use one of these toolchains.
       # On Fedora or Red Hat
       sudo dnf install arm-none-eabi-newlib
 
+#. Install a C library suitable for the target system.
```

Comment:
I think we should sort out how picolibc should be delivered so that these instructions can be more specific. Getting picolibc included in the standard Zephyr toolchain build would certainly make everything easier for users.

Reply:
Agreed. We already have the section `Other Cross Compilers`.

How about placing this in an independent section `Other C Libraries`
```
Other C Libraries
#################

It is possible to install another C library.

For example, you might install the picolibc-arm-none-eabi package
   on Debian-based Linux systems:

   .. code-block:: console

      # On Debian or Ubuntu
      sudo apt-get install picolibc-arm-none-eabi

.....
```
and then of course update the text with a bit more detailed information or link, to assist users on how to obtain picolib.
Of course, if/when picolib becomes part of Zephyr SDK, then it can just be referenced to Zephyr SDK.

Reply:
I've filed a PR with crosstools-ng to add picolibc support there; does that seem like a reasonable way to get picolibc support into the SDK?

 https://github.com/crosstool-ng/crosstool-ng/pull/1373